Tests/IntegrationTests/DropIn Tests/ComponentManagerTests.swift

```diff
@@ -11,6 +11,10 @@
 #if canImport(AdyenCashAppPay)
     @testable import AdyenCashAppPay
 #endif
+
+#if canImport(AdyenCashAppPay)
```

Comment:
twint

Reply:
Oh good catch, it was like that all this time